test.e2e/test-server.js

```diff
@@ -114,6 +115,15 @@ masterApp.post('/setup', function(req, res, next) {
           apiUrl: apiUrl,
           includeSchema: includeSchema,
         });
+      } else if (modelsToIgnore) {
```

Comment:
This way, `modelsToIgnore` and `includeSchema` cannot be use together.

I am proposing a different solution:

```js
if (includeSchema || modelsToIgnore) {
  // the new options-based API
  servicesScript = generator.services(lbApp, {
    ngModuleName: name,
    apiUrl: apiUrl,
    includeSchema: includeSchema,
    modelsToIgnore: modelsToIgnore,
  });
} else {
  // the old API...
}
```